src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -2197,10 +2195,10 @@ public override IOperation VisitConditionalAccess(IConditionalAccessOperation op
                 if (ITypeSymbolHelpers.IsNullableType(operation.Type) && !ITypeSymbolHelpers.IsNullableType(currentConditionalAccess.WhenNotNull.Type))
                 {
                     IOperation access = Visit(currentConditionalAccess.WhenNotNull);
-                    AddStatement(new FlowCapture(resultCaptureId, currentConditionalAccess.WhenNotNull.Syntax,
-                                                 TryMakeNullableValue((INamedTypeSymbol)operation.Type, access, compilation) ??
-                                                 // PROTOTYPE(dataflow): The scenario with missing constructor is not covered by unit-tests.
```

Comment:
This led to bunch of baseline updates for existing tests. I was unable to identify a case where `((INamedTypeSymbol)operation.Type).TypeArguments[0].Equals(access.Type) == false` , but have kept the `MakeInvalidOperation` invocation as a defensive approach.